modules/abstract-cosmos/src/lib/utils.ts

```diff
@@ -700,6 +700,42 @@ export class CosmosUtils implements BaseUtils {
     throw new NotImplementedError('validateAmount not implemented');
   }
 
+  /**
+   * Checks if a cosmos like Bech32 address matches given regular expression and
+   * validates memoId if present
+   * @param {string} address
+   * @param {RegExp} regExp Regular expression to validate the root address against after trimming the memoId
+   * @returns {boolean} true if address is valid
+   */
+  isValidCosmosLikeAddressWithMemoId(address: string, regExp: RegExp): boolean {
```

Comment:
seems to be same/similar to the other ones. can this live in a common place like abstract cosmos util since it isn't coin specific

Reply:
so atom is not part of abstract cosmos yet, so that's why it was separate. Resolving